plugins/woocommerce/changelog/2022-11-24-08-07-11-448089

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: enhancement
```

Comment:
Perhaps `patch` (backwards-compatible bug fix) and `tweak` (minor adjustment to the codebase) would be better here?

```suggestion
Significance: patch
Type: tweak
```